docker-compose.yml

```diff
@@ -14,6 +14,8 @@ services:
       - backend
 
   web:
+    ports:
+      - "127.0.0.1:8000:8000"
     environment:
```

Comment:
@lingxiaoyang if we are only using this binding in development mode, in the `docker-compose.override.yml` we don't relly need to take care of security and promiscuous ports because we are using the development server anyway.